Qubit Server/dev/src/main/java/org/labrad/qubits/Constants.java

```diff
@@ -35,7 +35,7 @@
   public static final String REGISTRY_SERVER = "Registry";
   public static final String HITTITE_SERVER = "Hittite T2100 Server";
   public static final String ANRITSU_SERVER = "Anritsu Server";
-  public static final String DC_RACK_SERVER = "DC Rack";
+  public static final String DC_RACK_SERVER = "DC Rack Server";
```

Comment:
I don't see the actual server name change either here or in the `ghz_fpga_server` changes; is this just not used currently?